internal/pgbackrest/reconcile.go

```diff
@@ -302,6 +303,14 @@ func addServerContainerAndVolume(
 		postgres.DataVolumeMount().Name: postgres.DataVolumeMount(),
 		postgres.WALVolumeMount().Name:  postgres.WALVolumeMount(),
 	}
+	if util.DefaultMutableFeatureGate.Enabled(util.TablespaceVolumes) {
+		for _, instance := range cluster.Spec.InstanceSets {
+			for _, vol := range instance.TablespaceVolumes {
+				tablespaceVolumeMount := postgres.TablespaceVolumeMount(vol.Name)
+				postgresMounts[postgres.TablespaceVolumeMount(vol.Name).Name] = tablespaceVolumeMount
```

Comment:
🤔 Am I reading this correctly? Can one call be reduced?
```suggestion
				tablespaceVolumeMount := postgres.TablespaceVolumeMount(vol.Name)
				postgresMounts[tablespaceVolumeMount.Name] = tablespaceVolumeMount
```